client/src/Booking.jsx

```diff
@@ -128,20 +132,32 @@ class Booking extends React.Component {
 
   render() {
     return (
-      <div >
-        <aside className="main-container">
-          <div className="overlay" className="overlay-gray">
-            <div className="booking-widget">
-              <div className="loading-overlay">
-                <h5 className="price-container">${this.state.price_per_night}</h5>
-                <span>per night (2 guests)</span>
+      <div className="booking">
+        <div className="container">
+          <div className="nightly-pricing-container">
+            <div className="content">
+              <h5 className="nightly-price">${this.state.price_per_night}
+              <br></br><span className="per">per night (2 guests)</span>
+              </h5>
+            </div>
+          </div>
+          <div className="dates-and-guests">
+            <div className="row">
+              <div className="col-xs-6 check-in-btn">
+                <CheckIn month={this.state.current_month} campId={this.state.campId} click={this.handleClick} submit={this.handleSubmit} />
```

Comment:
make new lines for components with many props:
 ```react.js
<CheckIn 
  month={this.state.current_month} 
  campId={this.state.campId}
  click={this.handleClick} {/* i think you mean onClick */}
  submit={this.handleSubmit} />
```

Reply:
ty done!